### PR TITLE
Updated : Automorphic number and Kaprekar number in C

### PR DESCRIPTION
**Pull request by Tausif121**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

I have added 2  updated programs in C on Automorphic number and Kaprekar number as assigned by @shraddhavp and i have made a separate PR as told by @yashaswibiyahut 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Related Issue: #1643 
Related PR: #1657

### CODE REVIEW

**Comment by shraddhavp** [comment]
@Tausif121  please add time complexities for the programs added

**Comment by Tausif121** [comment]
> @Tausif121 please add time complexities for the programs added

@shraddhavp , Time complexity added for both the programs.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Tausif121 Add your program link in `C/README.md` not in `C/math/README.md `. Also, format your code using code formatter.

**Comment by Tausif121** [comment]
> @Tausif121 Add your program link in `C/README.md` not in `C/math/README.md `. Also, format your code using code formatter.

@iamrajiv  Now i have updated C/README.md and also formatted my 2 prorams using code editor.
sorry for the mistake, this is my first code contribution

File: C/math/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-a
+# Programs added by Tausif
```
Comment (iamrajiv) [inline C/math/README.md]: @Tausif121 Do not edit this file.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

**Comment by Tausif121** [comment]
@iamrajiv , so sorry I just realised my mistake and thank you for helping.

**Comment by iamrajiv** [comment]
@all-contributors please add @Tausif121 for code

**Pull request merged**